manage

```diff
@@ -787,17 +787,21 @@ runTests() {
   rm ${BEHAVE_INI_TMP}
 
   # Export agent logs
-  mkdir -p .logs
-  docker logs acme_agent > .logs/acme_agent.log
-  docker logs bob_agent > .logs/bob_agent.log
-  docker logs faber_agent > .logs/faber_agent.log
-  docker logs mallory_agent > .logs/mallory_agent.log
-
-  if [[ "${USE_NGROK}" = "true" ]]; then
-    docker logs acme_agent-ngrok > .logs/acme_agent-ngrok.log
-    docker logs bob_agent-ngrok > .logs/bob_agent-ngrok.log
-    docker logs faber_agent-ngrok > .logs/faber_agent-ngrok.log
-    docker logs mallory_agent-ngrok > .logs/mallory_agent-ngrok.log
+  if [[ "${GITHUB_ACTIONS}" = "true" ]]; then
```

Comment:
How I read this is that it will only copy the logs if `GITHUB_ACTIONS == true`? Like the stament of `USE_NGROK`. 

Shouldn't this be inverted?

Reply:
Yes, yes it should...